source/UberRides/RideRequestButton.swift

```diff
@@ -119,12 +118,13 @@ import CoreLocation
         uberTitleLabel.textAlignment = .Right
     }
     
-    private func addTitle() {
+    private func configureTitleLabel() {
+        setSizesConfigs()
```

Comment:
I think this method call is unnecessary, you can just include `uberTitleLabel.textAlignment = .Right` and get rid of `setSizesConfigs()`